.github/workflows/build-and-publish-proto-bindings.yml

```diff
@@ -0,0 +1,53 @@
+name: Protobuf Bindings
+
+on: push
+#  pull_request:
+#  push:
+#    branches:
+#      - main
+
+jobs:
+  java_build:
+    name: Publishing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Version
+        run: |
+          # Strip git prefix from version.
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+
+          # Strip "v" prefix from tag name.
```

Comment:
Why are we stripping the v? We dont do that with the other release stuff, do we?

Reply:
No... the v is part of the normal go version number... we had some mixed uses before that caused issues so we try to keep the v everywhere.

Reply:
Sorry, I must have been working of an old example. 

Are we ok with including the `v` in the maven artifact? `io.provenance:proto-kotlin:v1.7.5`. Normally form maven it's just the version number `1.7.5`

Reply:
I am fine with following existing conventions if our standard doesn't make sense there